src/vsphere_cpi/spec/integration/set_vm_metadata_spec.rb

```diff
@@ -2,44 +2,100 @@
 require 'securerandom'
 require 'tempfile'
 require 'yaml'
+include LifecycleHelpers
```

Comment:
is it needed? I think it should work without this as the module is included in spec_helper.